src/main/java/spireMapOverhaul/zones/manasurge/ManaSurgeZone.java

```diff
@@ -206,11 +212,9 @@ public static void applyRandomPermanentModifier(AbstractCard card) {
     }
 
     public static void applyPermanentPositiveModifier(AbstractCard card) {
-        if (Math.random() < COMMON_CHANCE) {
-            int numberOfCommonModifiers = 4;
-            int selectedModifierIndex = (int) (Math.random() * numberOfCommonModifiers);
+        if (AbstractDungeon.cardRng.randomBoolean(COMMON_CHANCE)) {
             AbstractCardModifier modifier;
-            switch (selectedModifierIndex) {
+            switch (AbstractDungeon.cardRng.random(4)) {
                 case 0:
```

Comment:
Still would prefer these to be in lists so you don't have these rather long switch case monsters that blow up your lines of code. makes the code more maintanable.

Reply:
i think I've set up the lists correctly. haven't done something like this before so let's hope